app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java

```diff
@@ -229,35 +241,30 @@ private String chooseImageSource(final String thumbUrl, final Uri localUri) {
     /**
      * Retry upload when it is failed
      */
-    @OnClick(R.id.retryButton)
     public void retryUpload() {
         callback.retryUpload(contribution);
     }
 
     /**
      * Delete a failed upload attempt
      */
-    @OnClick(R.id.cancelButton)
     public void deleteUpload() {
         callback.deleteUpload(contribution);
     }
 
-    @OnClick(R.id.contributionImage)
     public void imageClicked() {
         callback.openMediaDetail(position, isWikipediaButtonDisplayed);
     }
```

Comment:
If this is only called from one place (the click listener) then you could probably inline and delete the method.